actnow/petitions/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+
+from .models import Petition, PetitionSignature
+
+admin.site.register(Petition)
```

Comment:
I think we can combine this to 
`admin.site.register([Petition, PetitionSignature])`